Rickytta/project-x
# [ch-167223633]Admin-dashboard

This PR will enable Admin to control property adverts 

#### Description of Task to be completed
I created HTML  CSS  designs and created a javascript .

#### How should this be manually tested?

Should be opened in browser,  After sign  sign in admin should be able to view the dashboard 

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167223633

#### Screenshots (if appropriate)```
